website/static/js/fangorn.js

```diff
@@ -1131,6 +1136,18 @@ var FGItemButtons = {
                     }, 'Delete'));
 
             }
+            if (item.data.permissions && item.data.permissions.view) {
+                rowButtons.push(
+                    m.component(FGButton, {
+                        onclick: function() {
+                            gotoFileEvent.call(tb, item);
+                        },
+                        tooltip: 'Open this file.',
+                        icon: 'fa fa-external-link',
+                        className : 'text-info'
+                    }, 'Open'));
```

Comment:
I'm going to change this to "View". "Open" typically means opening the file on the user's computer (that is the behavior on Dropbox, for example.